### PR TITLE
chore(funnels): Add breakdown attribution tooltip

### PR DESCRIPTION
**Pull request by neilkakkar**
## Problem

**Note: I'm changing global tooltips here to match what we have in figma. Let me know if this is not what we want!**

Change:
image

Without color change:

image

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/lib/components/Tooltip.tsx [hunk 1]
```diff
@@ -37,7 +37,11 @@ export function Tooltip({
     const child = React.isValidElement(children) ? children : <span>{children}</span>
 
     return props.title ? (
-        <AntdTooltip {...props} visible={isDefaultTooltip ? visible : localVisible && debouncedLocalVisible}>
+        <AntdTooltip
+            {...props}
+            visible={isDefaultTooltip ? visible : localVisible && debouncedLocalVisible}
+            color={'#2D2D2D'}
```
Comment (mariusandra) [inline frontend/src/lib/components/Tooltip.tsx:43]: Pretty sure we shouldn't hardcode colors into JSX, but do something cleaner... 🤔

Comment (neilkakkar) [reply frontend/src/lib/components/Tooltip.tsx:43]: It's a tooltip prop: https://ant.design/components/tooltip/

Any recommendations for making it cleaner?

And also, are we okay with making this global change? (cc: @clarkus )

Comment (mariusandra) [reply frontend/src/lib/components/Tooltip.tsx:43]: into global.scss

```css
.ant-tooltip { 
   background: var(--something-predefined);
}
```

Comment (neilkakkar) [reply frontend/src/lib/components/Tooltip.tsx:43]: Got it, thanks!

**Review by clarkus — CHANGES_REQUESTED** [review]
I don't think the content here is going to work for this context. Specifically the link to read more in the docs. These tooltips hide when the element being described is no longer hovered / is active. If a user were to move their cursor up to try to click on the link, or otherwise make the element lose focus, the tooltip would hide before they could click. 

The rest of the content seems like it will work. Can we increase the tooltip width to make the line lengths longer? It's a bit hard to read at this width.

**Comment by neilkakkar** [comment]
New look:

image

**Review by clarkus — APPROVED** [review]

**Pull request merged**